docs/api/api.md

```diff
@@ -0,0 +1,359 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# API doc of Manager API.
+Manager API directly operates ETCD and provides data management for APISIX, provides APIs for Front-end or other clients.
+
+**License:** [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+
+### /apisix/admin/check_ssl_cert
+
+#### POST
+##### Summary:
+
+verify SSL cert and key.
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| cert | body | cert of SSL | Yes | string |
+| key | body | key of SSL | Yes | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | SSL verify passed | [ApiError](#ApiError) |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/check_ssl_exists
+
+#### POST
+##### Summary:
+
+Check whether the SSL exists.
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| cert | body | cert of SSL | Yes | string |
+| key | body | key of SSL | Yes | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | SSL exists | [ApiError](#ApiError) |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/consumers
+
+#### GET
+##### Summary:
+
+Return the consumer list according to the specified page number and page size, and can search consumers by username.
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| page | query | page number | No | integer |
+| page_size | query | page size | No | integer |
+| username | query | username of consumer | No | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | list response | [ [consumer](#consumer) ] |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/notexist/routes
+
+#### GET
+##### Summary:
+
+Returns result of route exists checking by name and exclude id.
```

Comment:
I saw Imperative sentences are used in other places, so here we should also use it.

Reply:
fixed.